src/main/plugin-metadata/plugin-security.policy

```diff
@@ -1,3 +1,4 @@
 grant {
     permission java.lang.RuntimePermission "loadLibrary.KNNIndexV2_0_11";
+    permission java.net.SocketPermission "*", "connect,resolve";
```

Comment:
I see this changing the security policy. How will this impact k-NN during use? Do other plugins do this?

Reply:
We needed this to run security tests. Yes other plugins whitelist these.